include/internal/aes_platform.h

```diff
@@ -0,0 +1,391 @@
+/*
```

Comment:
This should be placed in crypto/include/internal.  The implication of placing a header file in include/internal is that it can be exported in whatever shared library, and there's no reason why libcrypto should export any symbol from here.

The providers that *use* modes will have to get crypto/include added to their INCLUDE

Reply:
Changed - paths seem to be ok.